src/core/qgsimagecache.cpp

```diff
@@ -491,6 +495,13 @@ QImage QgsImageCache::renderImage( const QString &path, QSize size, const bool k
   // render image at desired size -- null size means original size
   if ( !size.isValid() || size.isNull() || im.size() == size )
     return im;
+#if QT_VERSION >= QT_VERSION_CHECK(6, 8, 0)
+  else if ( im.format() == QImage::Format_CMYK8888 )
+  {
+    QgsMessageLog::logMessage( tr( "Impossible to scale a CMYK image : %1" ).arg( path ) );
+    return im;
```

Comment:
From a practical perspective, I don't think returning the unscaled image is very useful here. The result is going to be unusable in almost every case. Maybe we should force convert to RGB and then scale?

Reply:
Regarding my [comment](https://github.com/qgis/QGIS/pull/57417#issuecomment-2111783411), I propose to remove this

Reply:
Yep, +1 to remove